src/libs/BrickRoadsUtils.ts

```diff
@@ -0,0 +1,71 @@
+import Onyx, {OnyxCollection} from 'react-native-onyx';
+import {ValueOf} from 'type-fest';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import {Report} from '@src/types/onyx';
+import * as OptionsListUtils from './OptionsListUtils';
+import * as ReportActionsUtils from './ReportActionsUtils';
+import * as ReportUtils from './ReportUtils';
+
+let allReports: OnyxCollection<Report>;
+
+type BrickRoad = ValueOf<typeof CONST.BRICK_ROAD> | undefined;
+
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.REPORT,
+    waitForCollectionCallback: true,
+    callback: (value) => (allReports = value),
+});
+
+/**
+ * @param policyReport
+ * @returns BrickRoad for the policy passed as a param
+ */
+const getBrickRoadForPolicy = (policyReport: Report): BrickRoad => {
```

Comment:
I think that the policyReport name is not required here and it more confuses whether its some new term
```suggestion
const getBrickRoadForPolicy = (report: Report): BrickRoad => {
```